src/application/service/GetTests.ts

```diff
@@ -0,0 +1,16 @@
+import { TestRepository } from '../../domain/model/test/TestRepository';
+import { Test } from '../../domain/model/test/Test';
+import { TestId } from '../../domain/model/test/TestId';
+import { UserId } from '../../domain/model/user/UserId';
+
+export class GetTests {
+  constructor(private testRepository: TestRepository) {}
+
+  async byUserId(userId: string): Promise<Array<Test>> {
```

Comment:
Not sure if this should just be a getAll.  It's useful for us to have the userId in the path, because we are going to filter the tests based on the user's permissions.  Not sure of the best place for us to run the permission logic.  Should probably discus with Kostas as he has clearest vision on that.